### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by tombaker** [pypa/pypi-support#200]
Request for account recovery.

## PyPI Username

`tombaker`

---

##  Reason for request

Reason:

I successfully logged in today and changed my password.  My password vault tells me I entered my PyPI password on Sep 1, 2019 (which is when I started using the new password vault).  I'm quite sure that this was when I set up 2FA.  I still have the same device, with the same 2FA app, and it still generates codes for PyPI, but the Enter Authentication Form tells me that the TOTP codes are invalid.

In other words, I'm quite sure that 2FA was working for PyPI on Sep 1, 2019.

**Comment by tombaker** [comment]
According to my notes, I set up 2FA in August 2019.

**Comment by ewdurbin** [comment]
@tombaker PyPI passwords and TOTP are handled separately. Changing passwords shouldn't affect TOTP.

I note that a TOTP method was removed on 2019-09-01 and another added. Are you sure you don't have multiple records in your 2FA app?

**Comment by tombaker** [comment]
Hi Ernest,

Thank you for your email!

Last summer, when I was setting up 2FA, I did not understand that
authenticator apps are linked to specific devices.  I set up some accounts
using Google Authenticator on a tablet, but since I do not carry the tablet
around, I deleted the entries on my tablet and set up Authenticator on my
phone.  So what you say fits with that.  I thought I had used it to log
into PyPI successfully but I cannot swear that this was successful.  I do
have notes saying that I set i[ 2FA for PyPI on 2019-08-06 using Google
Authenticator.  I'm guessing that was on my tablet - the entry I later
deleted...

I'm sorry to bother you with this - it would seem clearly to be my fault.
I did not think it mattered on which device the Authencator app was running.

Some background: I am an amateur programmer, 62 years old, and after many
years of thrashing around and false starts my 'mklists' project is passing
over 150 tests and I have set up readthedocs for the documentation [1].  At
this rate, I'd say there is no great urgency to get PyPI working again -
I'm just at Version 0.2 - but the old code there is very preliminary and
should at least be replaced, and I'm hoping that 'mklists' will be
installable within the next few months.

Tom

[1] https://mklists.readthedocs.io/en/latest/

**Comment by ewdurbin** [comment]
@tombaker were you able to log in using the codes from your phone?

**Comment by tombaker** [comment]
No, those were the codes that were rejected.

**Comment by ewdurbin** [comment]
OK @tombaker I'll be in contact shortly to the primary email address associated with your PyPI account to proceed with account recovery.

**Comment by ewdurbin** [comment]
Confirmation complete, 2FA has been disabled and a password reset initiated. After completing the password reset you should regain access to your account.
